frontend/package.json

```diff
@@ -70,8 +71,13 @@
     "eslint-plugin-prettier": "^5.2.1",
     "eslint-plugin-react-hooks": "^4.6.2",
     "eslint-plugin-react-refresh": "^0.4.10",
+    "fs": "^0.0.1-security",
     "globals": "^15.9.0",
+    "lighthouse": "^12.2.1",
     "msw": "^2.3.5",
+    "path": "^0.12.7",
+    "playwright": "^1.48.2",
```

Comment:
This is a shortcoming of the `playwright-lighthouse` readme documentation, but this `playwright` installation is not needed because it is already installed as a part of `@playwright/test`.